From 7aa8ab10f304c74580526d77f392d76260f2f1a0 Mon Sep 17 00:00:00 2001
From: Simon Bruder <simon@sbruder.de>
Date: Fri, 22 Oct 2021 00:20:35 +0200
Subject: [PATCH] Load config from unprefixed location

This is required for distributions packaging SARndbox, because they
often install packages to a temporary location that differs from the
location the file will be available at when installed.
---
 makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/makefile b/makefile
index caf04b4..c200e61 100644
--- a/makefile
+++ b/makefile
@@ -98,7 +98,7 @@ config: Configure-End
 .PHONY: Configure-Begin
 Configure-Begin:
 	@cp Config.h Config.h.temp
-	@$(call CONFIG_SETSTRINGVAR,Config.h.temp,CONFIG_CONFIGDIR,$(ETCINSTALLDIR))
+	@$(call CONFIG_SETSTRINGVAR,Config.h.temp,CONFIG_CONFIGDIR,/$(CONFIGDIR))
 	@$(call CONFIG_SETSTRINGVAR,Config.h.temp,CONFIG_SHADERDIR,$(SHAREINSTALLDIR)/Shaders)
 	@if ! diff Config.h.temp Config.h > /dev/null ; then cp Config.h.temp Config.h ; fi
 	@rm Config.h.temp
-- 
2.31.1

